cmd/kubeadm/app/util/etcd/etcd_test.go

```diff
@@ -351,3 +352,141 @@ func TestGetRawEtcdEndpointsFromPodAnnotationWithoutRetry(t *testing.T) {
 		})
 	}
 }
+
+type fakeEtcdClient struct {
```

Comment:
we can move this near the top of the file.

Reply:
Fixed by https://github.com/kubernetes/kubernetes/pull/117792/commits/ee828f7c90caa9b25edd7499997d8a3d50478105